flume/README.md

```diff
@@ -342,6 +342,8 @@ The typical configuration when using the `HTTPSource`, the `OrionRestHandler`, t
 
 More advanced configurations can be found at [`doc/operation/performance_tuning_tips.md`](doc/operation/performance_tuning_tips.md).
 
+Kerberos authentication enabling in HDFS is described at [`doc/operation/hdfs_kerberos_authentication.md`](doc/operation/hdfs_kerberos_authentication.md).
```

Comment:
Although most of the detais on configuring Kerberos auth are in the hdfs_kerberos_authentication.md document, I think that at least we should include in the main README.md the following:

"If you are not using Kerberos, just set `cygnusagent.sinks.hdfs-sink.krb5_auth` to `false` and forgot about this thing :)"

or a similar statement.